### PR TITLE
updated README and dockerfile entrypoints to include -g daemon off flag

### PR DESCRIPTION
**Pull request by cpoole**
updated readme and dockerfiles for entrypoint with -g deamon mode

### CODE REVIEW

**Comment by neomantra** [comment]
Looks good to me.  Thanks for the PR.   I'm going to comment on some tweaks to the README.  Please put on your tree then I'll merge.

File: README.md [hunk 1]
```diff
@@ -69,11 +69,13 @@ Usage
 If you are happy with the build defaults, then you can use the openresty image from the [Docker Hub](https://hub.docker.com/r/openresty/openresty/).  The image tags available there are listed at the top of this README.
 
 ```
-docker run [options] openresty/openresty:latest-trusty /usr/local/openresty/nginx/sbin/nginx
+docker run [options] openresty/openresty:latest-trusty"
```
Comment (neomantra) [inline README.md]: Remove the trailing quote.

File: README.md [hunk 2]
```diff
@@ -69,11 +69,13 @@ Usage
...
 ```
 
 *[options]* would be things like -p to map ports and -v to map volumes.
 
+The `-g "daemon off;"` flag is used in the entrypoint to keep the nginx daemon running after container creation, this flag can also be added to the nginx.conf and ommitted from the entrypoint.
```
Comment (neomantra) [inline README.md]: The `-g "daemon off;"` directive is used in the Dockerfile ENTRYPOINT to keep the Nginx daemon running after container creation.  This directive can also be added to the nginx.conf and omitted from the ENTRYPOINT.

To invoke with another ENTRYPOINT, for example the `resty` utility, invoke like so:
```
docker run [options] --entrypoint /usr/local/openresty/bin/resty openresty/openresty:latest-alpine [script.lua]
```

**Comment by neomantra** [comment]
Actually, I'm making other changes to README, so I'll do that.  I'll merge this sometime today.  Thanks again.

**Comment by cpoole** [comment]
sounds good, thanks for the help @neomantra

**Pull request merged**